### DISCUSSION TITLE
Support of 8-channel LORA

### DISCUSSION THREAD

**Discussion by michcfr** [arendst/Tasmota#21550] [Ideas]
Hello,
I would be interested by the integration of a 8 channel LoRa gateway based on the SX1302 or SX1303 chips. 
Some works has already integrated the SX1302/03 driver with the ESP32: https://github.com/lora-gateway/esxp1302.

Is there any plan to integrate it? what would be the effort to integrate the SX1302/03 driver with Tasmota?

Regards,
Michel

**Comment by arendst** [comment]
What's the use-case for the 8-channel LoRa (no LoRaWan!!) gateway?

Current Tasmota LoRa(Wan) implementation allows connection of single channel LoRa devices and some LoRaWan devices set to operate at a single channel either by their userinterface or by LoRaWan ADR.

**Comment by michcfr** [comment]
The use case is similar to Zigbee2Tasmota: proposes a lightweight LoRa gateway/bridge solution that sends/receives MQTT messages. It is a knd of LoRa2Tasmota bridge.
The device can also supports both standard LoRaWAN (connect as LORA gateway) or MQTT.

**Comment by arendst** [comment]
There is a big difference between a LoRa or LoRaWan gateway. Tasmota definitly won't supply a LoRaWan gateway simply as there are better solutions.

Currently it is already a LoRa and a LoRaWan bridge for single channel LoRa(Wan) devices sending any data over MQTT. See https://tasmota.github.io/docs/LoRa-and-LoRaWan-Bridge/

**Comment by michcfr** [comment]
To rephrase my request:
Is there a (easy) way to adapt the Tasmota for LoRa and LoRaWan bridge to support 8-channels SX1302/SX1303 chips?
Is there any plan to integrate it? what would be the effort to integrate the SX1302/03 driver with Tasmota?


Regards,
Michel

**Comment by Jason2866** [comment]
There is no plan to support 8 channel.
